server/src/test/java/org/elasticsearch/repositories/blobstore/BlobStoreRepositoryRestoreTests.java

```diff
@@ -104,7 +104,7 @@ public void testRestoreSnapshotWithExistingFiles() throws IOException {
                     shardRouting,
                     shard.shardPath(),
                     shard.indexSettings().getIndexMetaData(),
-                    null,
+                    null, null,
```

Comment:
Nit: This is probably better on a separate line like the rest of the parameters.